### PR TITLE
Add affiliation when enrich

### PR DESCRIPTION
**Pull request by enkhalifapro**

### CODE REVIEW

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
All other da-ds data sources already have affiliations support in [affs.go](https://github.com/LF-Engineering/da-ds/blob/master/affs.go) file and support for mass/bulk uploading new identities to SH DB in [ds.go](https://github.com/LF-Engineering/da-ds/blob/master/ds.go#L178) file. Your code will duplicate this, and it misses 5-step algorithm: https://github.com/LF-Engineering/da-ds/blob/master/affs.go#L246 (we should either use this code which supports everything that grimoire did - by talking to SH DB directly - to be fastest and using bulk inserts for identities, or we should use dev-affiliation-api) - we don't want a new affiliations handler that confuses me and duplicates functionalities we already have. cc @fayazg @1010sachin

**Review by code-sleuth — APPROVED** [review]
/lgtm

File: cmd/dads/dads.go [hunk 1]
```diff
@@ -197,6 +197,19 @@ func buildBugzillaManager(ctx *lib.Ctx) (*bugzilla.Manager, error) {
 
 	params.GapURL = ctx.GapURL
 
+	params.AffBaseURL = ctx.Env("AFFILIATIONS_API_BASE_URL")
+	params.ESCacheURL = ctx.Env("ES_CACHE_URL")
+	params.ESCacheUsername = ctx.Env("ES_CACHE_USERNAME")
+	params.ESCachePassword = ctx.Env("ES_CACHE_PASSWORD")
+	params.AuthGrantType = ctx.Env("AUTH0_GRANT_TYPE")
+
```
Comment (ajinkyan83) [inline cmd/dads/dads.go:205]: If possible, can remove empty lines like this.

**Review by ajinkyan83 — APPROVED** [review]
/lgtm

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**